src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/ProteinsList.tsx

```diff
@@ -24,33 +27,68 @@ import {
 } from 'src/content/app/entity-viewer/shared/helpers/entity-helpers';
 import { defaultSort } from 'src/content/app/entity-viewer/shared/helpers/transcripts-sorter';
 
+import { toggleExpandedProtein } from 'src/content/app/entity-viewer/state/gene-view/proteins/geneViewProteinsSlice';
+import { getExpandedTranscriptIds } from 'src/content/app/entity-viewer/state/gene-view/proteins/geneViewProteinsSelectors';
+
 import { Gene } from 'src/content/app/entity-viewer/types/gene';
+import { RootState } from 'src/store';
 
 import styles from './ProteinsList.scss';
 
 type ProteinsListProps = {
   gene: Gene;
+  expandedTranscriptIds: string[];
+  toggleExpandedProtein: (id: string) => void;
 };
 
 const ProteinsList = (props: ProteinsListProps) => {
+  const { search } = useLocation();
+  const proteinIdToFocus = new URLSearchParams(search).get('protein_id');
+
   const sortedTranscripts = defaultSort(props.gene.transcripts);
   const proteinCodingTranscripts = sortedTranscripts.filter(
     isProteinCodingTranscript
   );
 
+  useEffect(() => {
+    const hasExpandedTranscripts = !!props.expandedTranscriptIds.length;
+    // Expand the first transcript by default
+    if (
+      !hasExpandedTranscripts &&
+      !proteinIdToFocus &&
+      sortedTranscripts[0].product
+    ) {
+      props.toggleExpandedProtein(sortedTranscripts[0].product.stable_id);
+    }
+  }, []);
+
   const longestProteinLength = getLongestProteinLength(props.gene);
 
   return (
     <div className={styles.proteinsList}>
-      {proteinCodingTranscripts.map((transcript) => (
-        <ProteinsListItem
-          key={transcript.stable_id}
-          transcript={transcript}
-          trackLength={longestProteinLength}
-        />
-      ))}
+      {proteinCodingTranscripts ? (
```

Comment:
According to line 49 above (`proteinCodingTranscripts = sortedTranscripts.filter ...`), `proteinCodingTranscripts` are guaranteed to be at very least an empty array, which makes them always truthy; so the CircleLoader below isn't going to be rendered.